docs/DesignDocs/Attestation_API_Proposal.md

```diff
@@ -624,29 +600,52 @@ by `oe_get_report()`.
 - `OE_FORMAT_UUID_SGX_ECDSA_P256_QUOTE`: for raw SGX ECDSA quote generated
 by the Intel SGX SDK.
 
-For evidence data wrapped by a header with a format ID, custom claims buffer
+For evidence data in format `OE_FORMAT_UUID_SGX_LOCAL_ATTESTATION`
+or `OE_FORMAT_UUID_SGX_ECDSA_P256`, custom claims buffer
 (if not empty) is attached to the end of the evidence, the list of claims returned
 by `oe_verify_evidence()` will contain one claim named
 `OE_CLAIM_CUSTOM_CLAIMS` if the evidence data contains a non-empty custom
-claims buffer. For evidence data that is an OE report (for an SGX
-ECDSA quote) or a raw SGX ECDSA quote, since this evidence data embeds custom
-claims buffer in its SGX quote directly,
+claims buffer. For evidence data in format `OE_FORMAT_UUID_SGX_ECDSA_P256_REPORT`
+or `OE_FORMAT_UUID_SGX_ECDSA_P256_QUOTE`, since this evidence data embeds the
+custom claims buffer in its SGX quote directly,
 the list of claims will contain one claim named `OE_CLAIM_SGX_REPORT_DATA`.
 
 To allow test of `oe_verify_evidence()` verification of SGX ECDSA reports and
 raw SGX ECDSA quotes, `oe_get_evidence()` will be extended to take one of the
 two format IDs (`OE_FORMAT_UUID_SGX_ECDSA_P256_REPORT` and
 `OE_FORMAT_UUID_SGX_ECDSA_P256_QUOTE`), to generate evidence in the specified
-format (without a header containing the format ID). For both formats, the
+format. For both formats, the
 `custom_claims` data will be put in the `report_data` field in the SGX quote
 contained in the output evidence.
 
 Similarly, for EPID evidence generation, the `custom_claims` input will be
-used as the SGX quote `report_data` directly, and the output evidence data
-will be raw EPID quote. Raw EPID quotes are what existing SGX EPID-based
-solutions have been using, this design will help maintain backward
+used as the SGX quote `report_data` directly. This design is consistent with
+existing SGX EPID-based solutions behavior, and will help maintain backward
 compatibility.
 
+The table below shows the structure of the evidence data for all the
+SGX format IDs, as generated by an attester plugin or verified by a verifier
+plugin. For every listed format, `oe_get_evidence()` can wrap with
+an attestation header when its `wrap_with_header` parameter is set to `true`.
```

Comment:
Note, this is inconsistent with the function signature earlier in the doc now.

Reply:
Thanks! Fixed in the latest commit.